fixtures/martin/rush-scanner-1-led.json

```diff
@@ -0,0 +1,1299 @@
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Indexing"
+          }
+        },
+        {
+          "dmxRange": [10, 14],
+          "type": "WheelSlot",
+          "slotNumber": 3,
+          "comment": "indexing",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Indexing"
+          }
+        },
+        {
+          "dmxRange": [15, 19],
+          "type": "WheelSlot",
+          "slotNumber": 4,
+          "comment": "indexing",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Indexing"
+          }
+        },
+        {
+          "dmxRange": [20, 24],
+          "type": "WheelSlot",
+          "slotNumber": 5,
+          "comment": "indexing",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Indexing"
+          }
+        },
+        {
+          "dmxRange": [25, 29],
+          "type": "WheelSlot",
+          "slotNumber": 6,
+          "comment": "indexing",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Indexing"
+          }
+        },
+        {
+          "dmxRange": [30, 34],
+          "type": "WheelSlot",
+          "slotNumber": 7,
+          "comment": "indexing",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Indexing"
+          }
+        },
+        {
+          "dmxRange": [35, 39],
+          "type": "WheelSlot",
+          "slotNumber": 2,
+          "comment": "rotating",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Rotation"
+          }
+        },
+        {
+          "dmxRange": [40, 44],
+          "type": "WheelSlot",
+          "slotNumber": 3,
+          "comment": "rotating",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Rotation"
+          }
+        },
+        {
+          "dmxRange": [45, 49],
+          "type": "WheelSlot",
+          "slotNumber": 4,
+          "comment": "rotating",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Rotation"
+          }
+        },
+        {
+          "dmxRange": [50, 54],
+          "type": "WheelSlot",
+          "slotNumber": 5,
+          "comment": "rotating",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Rotation"
+          }
+        },
+        {
+          "dmxRange": [55, 59],
+          "type": "WheelSlot",
+          "slotNumber": 6,
+          "comment": "rotating",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Rotation"
+          }
+        },
+        {
+          "dmxRange": [60, 64],
+          "type": "WheelSlot",
+          "slotNumber": 7,
+          "comment": "rotating",
+          "switchChannels": {
+            "Gobo 1 Indexing / Rotation": "Gobo 1 Rotation"
+          }
+        },
+        {
+          "dmxRange": [65, 88],
+          "type": "WheelShake",
```

Comment:
The text on page 26 in the manual seems to imply that only the slot is shaking:

> Individual gobos on wheel 1 can be rotated clockwise and counter-clockwise,
and a gobo shake function is available. Any gobo can be projected statically.
Both gobo wheels can be rotated clockwise and counter-clockwise.

Please add `"isShaking": "slot"`